ultraplot/constructor.py

```diff
@@ -1107,6 +1107,8 @@ def Locator(locator, *args, discrete=False, **kwargs):
     ultraplot.axes.Axes.colorbar
     ultraplot.constructor.Formatter
     """  # noqa: E501
+
+    lon0 = kwargs.pop("lon0", None)
```

Comment:
Is it true that the geo axes never use this factor function for building locators?

Reply:
I see calls here 

https://github.com/Ultraplot/ultraplot/blob/ae0029761b8e0160acfc38277c611c21fd8d84dd/ultraplot/axes/geo.py#L335

and so am confused as to how this is going to result in correct results.

Reply:
Yes. Just removed it.  It is correct since I am setting the lon0 attribute in 1641. Passing it along it not necessary. Not very happy with the implementation but it is not clear to me exactly how the formatters are generated.

Reply:
I would rather push it to the `_LonAxis` class but the that would require some tooling on how `lon0` is retrieved.